physionet-django/user/views.py

```diff
@@ -330,8 +330,15 @@ def edit_emails(request):
             set_public_email(request, public_email_form)
 
         elif 'add_email' in request.POST:
-            add_email_form = forms.AddEmailForm(request.POST)
-            add_email(request, add_email_form)
+            if associated_emails.count() >= settings.ASSOCIATED_EMAILS_ALLOWED:
+                messages.error(
+                    request,
+                    f'You cannot add more than {settings.ASSOCIATED_EMAILS_ALLOWED} email addresses.'
+                    f' If you need to more emails, please contact {settings.CONTACT_EMAIL}.'
```

Comment:
Perhaps change to: `If you need to link more emails to your account, ...`

Reply:
My bad. Sorry about the sentence. Fixing it rn :)